Xamarin.Forms.Platform.Android/ExportRendererAttribute.cs

```diff
@@ -12,5 +12,9 @@ public ExportRendererAttribute(Type handler, Type target) : this(handler, target
 		public ExportRendererAttribute(Type handler, Type target, Type[] supportedVisuals) : base(handler, target, supportedVisuals)
 		{
 		}
+
+		public ExportRendererAttribute(Type handler, Type target, Type[] supportedVisuals, short priority) : base(handler, target, supportedVisuals, priority)
```

Comment:
I wouldn't create an overload for an optional property. attributes supports named parameter resolved to properties, and they supported that way before we were able to use named parameters elsewhere

Reply:
I made the changes to make the priority as internal as it can be

I also retargeted this PR to 4.2.0 since the idea now is that we aren't adding any new APIs and this is resolving a bug in the ability to replace all default material renderers